experiment/adscert/signer.go

```diff
@@ -22,6 +23,7 @@ func (ns *NilSigner) Sign(destinationURL string, body []byte) (string, error) {
 }
 
 func NewAdCertsSigner(experimentAdCertsConfig config.ExperimentAdsCert) (Signer, error) {
+	logger.SetLoggerImpl(&SignerLogger{})
```

Comment:
I don't see the `logger` being used anywhere. Is there going to be a separate PR that makes use of this logger to log messages? Or am I missing something here?